### DISCUSSION TITLE
When will version 0.6.0 of vector be available?

### DISCUSSION THREAD

**Discussion by baloodevil** [supabase/supabase#22092] [Questions]
I'm interested in the [indexing performance improvements](https://supabase.com/blog/pgvector-fast-builds) coming with `vector` extension version `0.6.0`.  Going to Supabase Dashboard &rarr; Database &rarr; Extensions &rarr; searching for `vector`, shows the latest version available is `0.4.0`

**Comment by GaryAustin1** [comment]
If you are on Supabase hosted you just need to upgrade to the latest Postgres version in infrastructure settings.
![image](https://github.com/supabase/supabase/assets/54564956/4a8db914-1167-482d-9cbc-f66f7b8ac26b)
